gateways/nmi/request_builders.go

```diff
@@ -47,6 +47,15 @@ func buildCaptureRequest(testMode bool, securityKey string, request *sleet.Captu
 	}
 }
 
+func buildVoidRequest(testMode bool, securityKey string, request *sleet.VoidRequest) *Request {
+	return &Request{
+		SecurityKey:     securityKey,
+		TestMode:        enableTestMode(testMode),
+		TransactionID:   &request.TransactionReference,
+		TransactionType: "void",
```

Comment:
nit: maybe consts at the top for this, auth, capture? Just to so we know where these values come from?

Reply:
Good call, moved